tests/integ/modin/test_apply_snowpark_python_functions.py

```diff
@@ -1,20 +1,42 @@
 #
 # Copyright (c) 2012-2025 Snowflake Computing Inc. All rights reserved.
 #
+from typing import Union, Any
 
 import math
 
 import modin.pandas as pd
 import numpy as np
 import pandas as native_pd
 import pytest
+from pandas._typing import Scalar
+from pandas.core.dtypes.common import is_numeric_dtype
 
-from snowflake.snowpark.modin.plugin._internal.apply_utils import trunc_util
 from tests.integ.modin.utils import assert_frame_equal, assert_series_equal
 from tests.integ.utils.sql_counter import sql_count_checker
 from snowflake.snowpark.functions import trunc, sin, _log10, log, desc, asc
 
 
+def trunc_with_scale(e: Scalar, scale: Union[Any, int, float] = 0) -> Union[Any, float]:
```

Comment:
```suggestion
def trunc_with_scale(e: Scalar, scale: Union[Any, int, float] = 0) -> float
```

I think it's always a `float`, but I'm not sure.

Reply:
Yes it should be a float after the division